backend/server.py

```diff
@@ -20,13 +22,26 @@ def __init__(self) -> None:
         self.SERVER_NAME = SERVER_NAME
         self.websocket_info_dict: Dict[WEBSOCKET_TEMP_TYPE, int] = {}
         self.redis = redis.Redis(host=REDIS_URL, port=REDIS_PORT, db=REDIS_DB,)
-        redis_pubsub_instance = self.redis.pubsub(ignore_subscribe_messages=True,)
+        atexit.register(self.shutdown_handler)
 
+        redis_pubsub_instance = self.redis.pubsub(ignore_subscribe_messages=True,)
         redis_pubsub_instance.subscribe(
             **{"websocket-IPC": self.websocket_ipc_handler, "flask-IPC": self.flask_ipc_handler,}
         )
         self._redis_pubsub_thread = redis_pubsub_instance.run_in_thread(sleep_time=0.5,)
 
+    async def _shutdown_helper(self, tasks: List[Task[None]]) -> None:
```

Comment:
Remove Type Annotations

Reply:
*figure out how to _properly_ annotate that!

Reply:
probably just with a comment